src/components/BusinessCard/BusinessCard.js

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
 import Card from 'react-bootstrap/Card';
+import Button from 'react-bootstrap/Button';
 
-const businessCard = props => {
-  <Card style={{ width: '18rem' }}>
-    <Card.Img variant="top" src="holder.js/100px180" />
+const businessCard = props => (
+  <Card style={{ width: '18rem' }} className="mx-auto">
+    <Card.Img />
```

Comment:
does this need a source?

Reply:
It does but do we have access to businesses' images in DB? I wasn't sure so I left it like this